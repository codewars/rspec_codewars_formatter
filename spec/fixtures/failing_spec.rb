RSpec.describe "Failing" do
  it "fails" do
    expect(false).to be_truthy
  end
end
