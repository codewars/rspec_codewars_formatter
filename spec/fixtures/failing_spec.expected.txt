
<DESCRIBE::>Failing

<IT::>fails

<FAILED::>expected: truthy value<:LF:>     got: false

<COMPLETEDIN::>15.6654

<COMPLETEDIN::>15.9464
